src/main/java/net/javadiscord/javabot/systems/help/HelpManager.java

```diff
@@ -107,6 +110,17 @@ public void close(IReplyCallback callback, boolean withHelpers, @Nullable String
 				.queue(s -> postThread.getManager().setLocked(true).setArchived(true).queue());
 	}
 
+	/**
+	 * Closes the {@link HelpManager#postThread}.
+	 *
+	 * @param closedBy    The {@link UserSnowflake} that closed this post.
+	 * @param reason      The reason for closing this post.
+	 */
+	public void close(UserSnowflake closedBy, @Nullable String reason) {
+		postThread.sendMessageEmbeds(buildPostClosedEmbed(closedBy, reason))
+				.queue(s -> postThread.getManager().setLocked(true).setArchived(true).queue());
```

Comment:
Shouldn't that still allow the OP to reopen it?
```suggestion
				.queue(s -> postThread.getManager().setArchived(true).queue());
```